func checkErr(err) {
  if err != nil {
    panic(err)
  }
}
